services/nginz/integration-test/conf/nginz/nginx.conf

```diff
@@ -274,11 +274,16 @@ http {
         proxy_pass http://brig;
     }
 
-    location /system/settings/unauthorized$ {
+    location ~* ^(/v[0-9]+)?/system/settings/unauthorized$ {
         include common_response_no_zauth.conf;
         proxy_pass http://brig;
     }
 
+    location ~* ^/system/settings$ {
```

Comment:
Why does `/system/settings/unauthorized` accept a version prefix and `/system/settings` not? :thinking: